app/page/template/list/start-ui.htm

```diff
@@ -114,6 +114,12 @@
   </div>
 
   <!-- ko if: showNoContacts() -->
+    <ul class="start-ui-create-group left-list-items">
+      <li class="left-list-item left-list-item-clickable" data-bind="click: clickOnCreateGroup" data-uie-name="go-create-group">
+        <group-icon class="left-column-icon"></group-icon>
+        <div class="center-column" data-bind="l10n_text: z.string.searchCreateGroup"></div>
+      </li>
+    </ul>
```

Comment:
Can we find a solution that does not require duplication of the html?

Reply:
The Andymator and I found a CSS solution to the problem. 👍